star-rx72n-firmware/tests/mocks/mock_uart_hw.c

```
@@ -527,7 +528,14 @@ rx_err_t uart_rx_available(uint8_t channel, bool* available)
 
 rx_err_t uart_init(void)
 {
-  return uart_init_channel(9, 115200, 0x0B, 7, 0x0B, 6);
+  const uart_channel_config_t config = {
```

Clang (14.0.6)

[warning] variable 'config' is not initialized

(cppcoreguidelines-init-variables)